openid-connect-4-credential-issuance-1_0.md

```diff
@@ -287,39 +295,119 @@ experience, it seems to be the better way to leave implementers the choice. If t
+
+The following claims are used in each object in the `credentials` property:
 
-Every credential object has the following properties:
+* `type`: OPTIONAL. A JSON string denoting the type of the requested credential. MUST be present if `manifest_id` is not present.
+* `manifest_id`: OPTIONAL. JSON String refering to a credentoal manifest published by the credential issuer. MUST be present if `type` is not present. 
+* `format`: OPTIONAL. A JSON string representing a format in which the credential is requested to be issued. Valid values defined by this specification are `jwt_vc` and `ldp_vc`. Profiles of this specification MAY define addtional format values.  
+* `vp_token`: OPTIONAL. A parameter defined in [@OIDC4VP] used to convey required verifiable presentations. The verifiable presentations passed in this parameter MUST be bound to a `p_nonce` generated by the respective issuer from the Nonce Endpoint. 
+* `presentation_submission`: OPTIONAL. JSON object as defined in [@DIF.CredentialManifest]. This object refers to verifiable presentations required for the respective credential accoridng to the Credential Manifest and provided in an authorization request. All entries in the `descriptor_map` refer to verifiable presentations provided in the `vp_token` authorization request parameter.  
 
-* `type`: REQUIRED. A JSON string denoting the type of the requested credential.
-* `format`: OPTIONAL. A JSON string representing a format in which the credential is requested to be issued. 
-Valid values defined by this specification are `jwt_vc` and `ldp_vc`. Profiles of this specification MAY define 
-addtional format values.  
+Note that `type` and `format` are used in a dynamic approach of presenting input credentials to the Issuer, when the Client has not pre-obtained a Credential Manifest. `manifest_id` and `presentation_submission` are used in a static approach of presenting input credentials to the Issuer, when the Client has pre-obtained a Credential Manifest. These two approaches MAY be combined in one request.
 
-Note: passing the format to the authorization request is informational and allows the credential issuer to 
-refuse early, in case it does not support the requested format/credential combination. The client MAY request 
-issuance of credentials in other formats as well later in the process.
+Note that passing the format to the authorization request is informational and allows the credential issuer to refuse early, in case it does not support the requested format/credential combination. The client MAY request issuance of credentials in other formats as well later in the process.
+
+Note that `credential_application` defined in [@DIF.CredentialManifest] is not required by this specification.
+
+#### Example of an Authorization Request with a dynamic approach of presenting input credentials
```

Comment:
This is just a simple example of requesting authz for credential issuance. Why such a complicated sub section title?

Reply:
I wanted to highlight that it is an example that uses `type` and not `manifest_id` - made it simpler